.github/workflows/publish_nuget.yml

```
@@ -4,6 +4,8 @@ on:
   push:
     branches:
     - 'master'
+  pull_request_target:
+    types: [synchronize]
```

Should it not also trigger when we open a PR?

```suggestion
    types:
      - opened
      - synchronize
```

---

I didn't know "opened" wasn't a type of "synchronize".